src/MCPServer/lib/assets/workflow.json

```diff
@@ -902,7 +902,35 @@
                 "replacements": [
                     {
                         "description": {
-                            "en": "5 - normal compression mode",
+                            "en": "1 - fastest compression",
+                            "es": "1 - el modo más rápido",
+                            "fr": "1 - mode rapide",
+                            "ja": "1 - 最高速モード",
+                            "pt_BR": "1 - modo mais rápido",
+                            "sv": "1 - snabbaste läget"
+                        },
+                        "id": "ecfad581-b007-4612-a0e0-fcc551f4057f",
+                        "items": {
+                            "AIPCompressionLevel": "1"
+                      	}
+                    },
+                    {
+                      	"description": {
+	                          "en": "3 - fast compression",
+	                          "es": "3 - mode de compresión rápido",
+	                          "fr": "3 - mode de compression rapide",
+	                          "ja": "3 - 高速圧縮モード",
+	                          "pt_BR": "3 - modo de compressão rápida",
+	                          "sv": "3 - snabbkomprimeringsläge"
+                      	},
+	                      "id": "85b2243e-ff97-4ca8-80e8-3c6b0842b360",
+	                      "items": {
+	                          "AIPCompressionLevel": "3"
+	                      }
+	                  },
```

Comment:
This block has indentation issues, e.g. L918 has a tab, other lines seem to be off by some spaces.